src/client/package.json

```diff
@@ -81,10 +81,7 @@
     "react-window": "^1.8.8",
     "ress": "^3.0.0",
     "rxjs": "^7.5.4",
-    "styled-components": "^5.2.2",
-    "typeface-lato": "^1.1.13",
-    "typeface-montserrat": "^1.1.13",
-    "typeface-roboto": "^1.1.13"
+    "styled-components": "^5.2.2"
```

Comment:
idle thought: we should look at later [styled-components](https://www.npmjs.com/package/styled-components/v/6.0.0-beta.15)